compiler/luci/pass/src/RemoveRedundantTranspose.test.cpp

```diff
@@ -138,15 +255,67 @@ TEST(RemoveRedundantTransposePass, remove_consecutive_transpose_function_type2)
   while (pass.run(graph.get()))
     ;
   luci::CircleTranspose *transpose_node = nullptr;
+  int count = 0;
+  for (auto node : loco::active_nodes(loco::output_nodes(graph.get())))
+  {
+    auto trans = dynamic_cast<luci::CircleTranspose *>(node);
+    if (not trans)
+      continue;
+    transpose_node = trans;
+    count++;
+  }
+  // Just one transpose node, with updated perm constant.
+  ASSERT_EQ(1, count);
```

Comment:
If we need to check the number of `CircleTranspose` then checking number of `count` is needed.
But here we only check the value is 1. Then I think `ASSERT_NE(nullptr, transpose_node);` is enough.
Is there any other reason for adding `count` that I may not thought of?

Reply:
Oh, yes. Just one op, so I also think remove this would be better. thanks.